pandas/core/indexes/base.py

```diff
@@ -4281,9 +4288,9 @@ def _maybe_cast_indexer(self, key):
 
     def _validate_indexer(self, form, key, kind):
         """
-        if we are positional indexer
+        If we are positional indexer
         validate that we have appropriate typed bounds
-        must be an integer
+        must be an integer.
```

Comment:
If you don't mind making the lines go until close to 79 characters, I think this fits in two lines.